pet001/dataset_description.json

```diff
@@ -0,0 +1 @@
+{"BIDSVersion":"1.0.2","License":"CCO license","Name":"[11C]CIMBI36 PET dataset of a pig","Authors":["Melanie Ganz-Benjaminsen","Martin Noergaard","Hanne Demant Hansen"],"Acknowledgements":"Knudsen GM, Jensen PS, Erritzoe D, Baaré WFC, Ettrup A, Fisher PM, Gillings N, Hansen HD, Hansen LK, Hasselbalch SG, Henningsson S, Herth MM, Holst KK, Iversen P, Kessing LV, Macoveanu J, Madsen KS, Mortensen EL, Nielsen FÅ, Paulson OB, Siebner HR, Stenbæk DS, Svarer C, Jernigan TL, Strother SC, Frokjaer VG. The Center for Integrated Molecular Brain Imaging (Cimbi) Database. NeuroImage. 2016 Jan 1;124(Pt B):1213-1219","HowToAcknowledge":"This data was obtained from the Cimbi database.","Funding":["Lundbeck Foundation R90-A7722","Danish Research Council 09-063598","Rigshospitalet"],"DatasetDOI":""}
```

Comment:
could you please format this JSON (and the other dataset_description JSON files in this PR) so that it is human readable?

you are already doing that for most of the other JSONs, but not for all of them.

example of a good one (screenshot):

![image](https://user-images.githubusercontent.com/9084751/113004389-de31bc00-9173-11eb-97f0-7bb16b3a824a.png)